### PR TITLE
kill memcached in favour of redis

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/style/static/style/lib/bootstrap-3.2.0 [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 6fe9cd9fef3e0960d0ef8484dea7aecd0880de84
+Subproject commit a7747dedf24b1a7245bc3088f265280873acd627
```
Comment (czue) [inline corehq/apps/style/static/style/lib/bootstrap-3.2.0]: it's likely fine but i'm guessing you didn't mean to include this?

**Comment by czue** [comment]
looks good - i'm signed off when QA passes

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by snopoke** [comment]
@czue :white_check_mark:

**Comment by czue** [comment]
whee

**Pull request merged**